esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2189,6 +2189,12 @@ bool CFileSprayEx::onSprayVariable(IEspContext &context, IEspSprayVariable &req,
         if (!req.getExpireDays_isNull())
             options->setExpireDays(req.getExpireDays());
 
+        if (req.getKeepSourceEncoding_isNull())
+            // Set default value
+            options->setKeepSourceEncoding(true);
```

Comment:
Should remove this afaics. It shouldn't be necessary to set this
It will not set for older eclwatch clients who create DFU workunits (that don't have this code),
but that's fine, because in DFU where this option is read, it defaults to true if not set.

, it should assume (default) keepSourceFormatting, so

Reply:
@AttilaVamos - discussed offline, but I not sure this is right.

IDFUWorkunit::getKeepSourceEncoding() return true by default (if DFU workunit option is not set).
That makes sense - that is the default wanted, for new and legacy workunits.

If the request is isNull(), then this code should not set the DFU workunit option.. i.e. it will be the same as legacy workunits, without the setting set, and default to true via IDFUWorkunit::getKeepSourceEncoding()

I suspect the code here should look like:
```
if (!req.getKeepSourceEncoding_isNull() && !req.getKeepSourceEncoding())
    options->setKeepSourceEncoding(false);
```

Meaning if request is non null and it is false, set it into the DFU workunit option.
Otherwise leaving it unset, means DFUWorknit::getKeepSourceEncoding() will return true by default as desired.